src/providers/bedrock/chatComplete.ts

```diff
@@ -830,176 +245,103 @@ export const BedrockAI21ChatCompleteResponseTransform: (
-    const completion_tokens =
-      Number(responseHeaders.get('X-Amzn-Bedrock-Output-Token-Count')) || 0;
-    return {
-      id: response.id.toString(),
+  if ('output' in response) {
+    const responseObj: ChatCompletionResponse = {
+      id: Date.now().toString(),
       object: 'chat.completion',
       created: Math.floor(Date.now() / 1000),
       model: '',
       provider: BEDROCK,
-      choices: response.completions.map((completion, index) => ({
-        index: index,
-        message: {
-          role: 'assistant',
-          content: completion.data.text,
-        },
-        finish_reason: completion.finishReason?.reason,
-      })),
-      usage: {
-        prompt_tokens: prompt_tokens,
-        completion_tokens: completion_tokens,
-        total_tokens: prompt_tokens + completion_tokens,
-      },
-    };
-  }
-
-  return generateInvalidProviderResponseError(response, BEDROCK);
-};
-
-interface BedrockAnthropicChatCompleteResponse {
-  id: string;
-  type: string;
-  role: string;
-  content: AnthropicContentItem[];
-  stop_reason: string;
-  model: string;
-  stop_sequence: null | string;
-}
-
-export const BedrockAnthropicChatCompleteResponseTransform: (
-  response: BedrockAnthropicChatCompleteResponse | BedrockErrorResponse,
-  responseStatus: number,
-  responseHeaders: Headers
-) => ChatCompletionResponse | ErrorResponse = (
-  response,
-  responseStatus,
-  responseHeaders
-) => {
-  if (responseStatus !== 200) {
-    const errorResposne = BedrockErrorResponseTransform(
-      response as BedrockErrorResponse
-    );
-    if (errorResposne) return errorResposne;
-  }
-
-  if ('content' in response) {
-    const prompt_tokens =
-      Number(responseHeaders.get('X-Amzn-Bedrock-Input-Token-Count')) || 0;
-    const completion_tokens =
-      Number(responseHeaders.get('X-Amzn-Bedrock-Output-Token-Count')) || 0;
-
-    let content = '';
-    if (response.content.length && response.content[0].type === 'text') {
-      content = response.content[0].text;
-    }
-
-    let toolCalls: any = [];
-    response.content.forEach((item) => {
-      if (item.type === 'tool_use') {
-        toolCalls.push({
-          id: item.id,
-          type: 'function',
-          function: {
-            name: item.name,
-            arguments: JSON.stringify(item.input),
-          },
-        });
-      }
-    });
-
-    return {
-      id: response.id,
-      object: 'chat.completion',
-      created: Math.floor(Date.now() / 1000),
-      model: response.model,
-      provider: BEDROCK,
       choices: [
         {
+          index: 0,
           message: {
             role: 'assistant',
-            content,
-            tool_calls: toolCalls.length ? toolCalls : undefined,
+            content: response.output.message.content
+              .filter((content) => content.text)
+              .reduce((acc, content) => acc + content.text + '\n', ''),
+            tool_calls: response.output.message.content
```

Comment:
Due to this logic, the response will always have tool_calls as empty array when the response does not have any tool calls. Can we make it optional?